src/utils/helpers.ts

```diff
@@ -0,0 +1,16 @@
+export function listToMatrix(list, elementsPerSubArray) {
```

Comment:
i think you can improve your variable names to be more descriptive and also add types to the parameters.